src/server/controllers/user/signUp.js

```diff
@@ -1,11 +1,12 @@
 import { nanoid } from 'nanoid';
-import { sendEmail } from '../handlers/sendEmail.js';
-import { User } from '../models/schema/users/userSchema.js';
+import { User } from '#schemas/users/userSchema.js';
+import { getUserByEmail } from '#handlers/userHelpers.js';
```

Comment:
We can even make handler for creating `User` 

Implement function to create new user and import it from same file same like `getUserByEmail`

f.ex.
`import { getUserByEmail, createNewUser } from '#handlers/userHelpers.js';`
...
`const user = createNewUser(email)`

Reply:
❓

Reply:
Done that also!